docs/_blog/ldk-an-sdk-for-the-lightning-network.md

```diff
@@ -0,0 +1,88 @@
+
+We created LDK in multiple languages with an API-first approach designed to run at the application layer, like Rust’s [Persist](https://docs.rs/lightning/latest/lightning/chain/chainmonitor/trait.Persist.html) trait. Persist defines behaviour for persisting channel states but lets you specify whether you write this data to disk or another backup mechanism, such as the cloud. You don’t need to write an LN implementation from scratch or modify an existing one to use LN functionality. Just call our APIs from your app.
+
+Finally (for this section), LDK’s flexibility enables several different architectures without sacrificing security. Its lightweight design can be optimized to run on embedded devices or HSMs (hardware security modules) and it doesn’t make system calls, so it can run in almost any OS environment. For example, you can opt to run some Lightning logic, such as signing transactions and updating channel states on an HSM that has specific [spending policies](https://gitlab.com/lightning-signer/docs/-/blob/master/README.md) and manages private keys. Then you connect it to a server with its own TCP/IP stack using a serial communication method such as USB. 
+
+More on LDK use cases [here](https://lightningdevkit.org/introduction/use_cases/).
+
+## Who uses LDK?
+
+LDK supports applications that require running many nodes (such as server/custodial apps), letting users run nodes on mobile devices, integrating with existing infrastructure, broad customization options, and resource-constrained embedded systems such as point of sale, IoT, and so on and so on, etc.  
+
+[Blue Wallet](https://bluewallet.io) was LDK’s patient zero and ideal first adopter since it is both non-custodial and mobile. Their team maintains [rn-ldk](https://github.com/BlueWallet/rn-ldk) and runs it on both Android and iOS.The wallet is likely to ship with LDK soon but not before some critical infrastructure is in place. While non-custodial mobile wallets are LDK’s primary use case, that doesn’t make it limited to them.
+
+[CashApp](https://cash.app), one of the world’s most used payment apps, chose LDK for its Lightning implementation because of its customizability. While custodial server-based solutions are not the primary use case for LDK, it shows how adaptable the library is. Ryan Loomba recently explained some of the trade-offs their team had to think about when comparing LDK to other solutions [at bitcoin++](https://www.youtube.com/watch?v=5YmfEgh-LC8&t=2442s).
```

Comment:
Should we link the Spiral talk somewhere? Or is there a concern with sound quality?

Reply:
Or maybe not the right blob post for it?

Reply:
Wait which Spiral talk?

Reply:
Ah, I meant the LDK talk by Sprial (Arik and me).

Reply:
Oh okay sure, will add that in 💯